### ISSUE TITLE
New --data option question

### ISSUE THREAD

**Issue by mjablecnik** [xnbox/DeepfakeHTTP#12]
Hello @xnbox,
I see that you added a new --data option into DeepfakeHTTP but I didn't found any example of how to use it..
Can you write here some description of this new functionality please? I tried understand it but I am not sure if I understand it right.
Can you write here also some small example? Thank you very much for your work. :-)

**Comment by xnbox** [comment]
Hi Martin,
The documentation is not ready yet.
Very dirty example:

<code>mydata.json</data>

```json
{
    "id": 28,
    "address" : {
        "city": "AAAA",
        "country": "BBB"
    },
    "kkk": [100, 200, 300, 400, 500, 600, 700]
}
```

```text
POST /action_page.php HTTP/1.1
Content-Type: application/x-www-form-urlencoded

HTTP/1.1 200 OK
Content-Type: text/html
X-Body-Type: text/template

<!DOCTYPE html>
<html lang="en">
<body>
    <h1>Hello, ${address.city} ${address.country}!</h1> <!-- print: Hello, AAA BBB -->
 ${kkk[1]} <!-- print: 200 -->
 ${kkk[$]} <!-- print: some random value from kkk array -->
</body>
</html>
```
Thanks for your support!!

**Comment by mjablecnik** [comment]
Thank you very much for your response.
I am trying your example but I gain this Error response:
![image](https://user-images.githubusercontent.com/2686906/133896972-0c7bb724-5029-46e4-9e44-0862eafdd8ca.png)

Are you sure that you sent me the right working example?

**Comment by xnbox** [comment]
Oh, I see! Bug here.
Fixed in v1.0.14

**Comment by mjablecnik** [comment]
Yes now it works right now. ;-) Is it also possible to add this variables also into request param or request body?
For example:
```
POST /action_page_${address.city}.php HTTP/1.1
Content-Type: application/x-www-form-urlencoded
```
And call it as: 
```
curl -X POST http://localhost:3000/action_page_AAAA.php
```
Then I will can have all values in the separate file and use only variables..

**Comment by xnbox** [comment]
I think this is a great idea!
Thank you!

**Comment by xnbox** [comment]
implemented in v1.0,15
